schema/workbook.json

```diff
@@ -1,2 +1,754 @@
+                },
+                "styleSettings": {
+                    "$ref": "#/definitions/styleSettings"
+                },
+                "conditionalVisibility": {
+                    "$ref": "#/definitions/conditionalVisibility"
+                }
+            }
+        },
+        "link": {
+            "$id": "#/definitions/link",
+            "type": "object",
+            "required": [
+                "type",
+                "content"
+            ],
+            "properties": {
+                "type": {
+                    "const": 11
+                },
+                "content": {
+                    "type": "object",
+                    "properties": {
+                        "version": {
+                            "type": "string",
+                            "description": "Version of link step",
+                            "examples": [
+                                "LinkItem/2.0"
+                            ]
+                        },
+                        "style": {
+                            "type": "string",
+                            "description": "style of link",
+                            "examples": [
+                                "bullets",
+                                "paragraph",
+                                "navigation"
+                            ]
+                        },
+                        "links": {
+                            "type": "array",
+                            "items": [
+                                {
+                                    "$ref": "#/definitions/linkItem"
+                                }
+                            ]
+                        }
+                    }
+                },
+                "name": {
+                    "$ref": "#/definitions/name"
+                },
+                "customWidth": {
+                    "$ref": "#/definitions/customWidth"
+                },
+                "showPin": {
+                    "$ref": "#/definitions/showPin"
+                },
+                "styleSettings": {
+                    "$ref": "#/definitions/styleSettings"
+                },
+                "conditionalVisibility": {
+                    "$ref": "#/definitions/conditionalVisibility"
+                }
+            }
+        },
+        "parameter": {
+            "$id": "#/definitions/parameter",
+            "type": "object",
+            "required": [
+                "type",
+                "content"
+            ],
+            "properties": {
+                "type": {
+                    "const": 9
+                },
+                "content": {
+                    "type": "object",
+                    "properties": {
+                        "version": {
+                            "type": "string",
+                            "description": "Version of parameter step",
+                            "examples": [
+                                "KqlParameterItem/2.0"
```

Comment:
check on the versions, i think all of them except metrics should be default `whatever/1.0` ? instead of examples?